ftw/simplelayout/browser/resources/integration.theme.scss

```diff
@@ -75,3 +75,45 @@
 .contenttreeWindow {
     z-index: 70001 !important;
 }
+
+// Listingblock
+.sl-block.ftw-simplelayout-filelistingblock {
+  table {
+    tr {
+      th {
+        white-space: nowrap;
+        text-align: left;
+        &.header-getObjSize {
+          text-align: right;
+        }
+      }
+      td {
+
+        &.column-getContentType {
+          width: 16px;
+        }
+
+        &.column-modified {
+          width: 85px;
+        }
+
+        &.column-portal_type {
```

Comment:
In the screenshots I can see a column named `type`. But `column-getContentType` or `column-portal_type` suggest a column for displaying a portal type. So which of these classes refer to the screenshot and what about the class which is not in the screenshot.